packages/ra-core/src/core/useGetRecordRepresentation.spec.tsx

```diff
@@ -93,4 +93,67 @@ describe('useRecordRepresentation', () => {
         );
         screen.getByText('Hello');
     });
+    it('should return the record name at first', () => {
+        render(
+            <UseRecordRepresentation
+                resource="users"
+                record={{
+                    name: 'Ipsum',
+                    title: 'Lorem',
+                    label: 'lorem-ipsum',
+                    reference: '456',
+                    id: '123',
+                    author: 'John Doe',
+                }}
+            />
+        );
+        screen.getByText('Ipsum');
+    });
+    it('should return the record title at second', () => {
+        render(
+            <UseRecordRepresentation
+                resource="users"
+                record={{
+                    title: 'Lorem',
+                    label: 'lorem-ipsum',
+                    reference: '456',
+                    id: '123',
+                    author: 'John Doe',
+                }}
+            />
+        );
+        screen.getByText('Lorem');
+    });
+    it('should return the record label at second', () => {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why are there two "second"? :joy: